README.md

```diff
@@ -1,6 +1,8 @@
 # eslint-plugin-vue-libs
 
-ESLint plugin for libs in the vuejs organization. Note this is an internal plugin/config for the development of Vue itself, not intended for Vue app development.
+> ESLint plugin for libs in the vuejs organization.
+
+❗️ Note this is an internal plugin/config for the development of Vue itself, not intended for Vue app development.
```

Comment:
I think github emoji code is human editable rather than emoji binary code. (e.g: `:exclamation:`)